### ISSUE TITLE
Ability to merge rooms

### ISSUE THREAD

**Issue by ara4n** [vector-im/vector-web#2299]
I've ended up with two almost identical rooms active at different times. I wish i could merge them.

**Comment by betz** [comment]
interesting

**Comment by ara4n** [comment]
Just realised (thanks to strk in HQ) that a simple solution to this might be to just go and finally implement a 'redirect' state event for a room, which an admin can set on a room to advise all clients to go autojoin an alternative one, to merge two together - similar to IRC channel redirects that many IRCd support.

I am confused that we've not got a bug for this before.  Going to open a spec bug, given that's really where this stuff should be being tracked.

**Comment by t3chguy** [comment]
tombstones could probably be used for this now

**Comment by xpseudonym** [comment]
Is this conceptually a dup of https://github.com/vector-im/riot-web/issues/722
?

**Comment by martindale** [comment]
> tombstones could probably be used for this now

Yes, has been mentioned before — would be useful to see in the UI still
